modules/nf-core/mashmap/main.nf

```diff
@@ -0,0 +1,48 @@
+process MASHMAP {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/mashmap:3.1.3--h07ea13f_0':
+        'biocontainers/mashmap:3.1.3--h07ea13f_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+    path(reference)
```

Comment:
We generally have a second meta map on the reference genome, to support tools/pipelines which have more than one reference genome involved.

Reply:
Ah, alright. I've added meta2 now.